cirq/circuits/circuit_operation_test.py

```diff
@@ -38,6 +38,19 @@ def test_properties():
     assert op == circuit.to_op()
 
 
+def type_validated():
```

Comment:
Need to call this `test_*` for pytest to pick it up as a test. Maybe call it `test_circuit_type`.

Reply:
Thanks for catching this!